data/human/navy missions intro.txt

```diff
@@ -0,0 +1,1496 @@
+			`He sighs. "Sorry. These days, it feels like I can never quite keep up with my orders and it's starting to weigh on me," he says glumly. "Come on, let's get this shit loaded onto your ships," he adds, fumbling for a new cigarette.`
+				goto end
+			label incident
+			`The dockworker stops dead in his tracks at your genuine honesty. "Wait...you're serious?"`
+			`	"I give you my word," you reply, nodding. Only then does the dockworker realize what a fool he's made of himself, glancing left and right with a cowed look.`
+			`	"I don't know if I can," he laments, "I don't want to risk losing my job."`
+			`	"Trust me," you reply gently, "whatever you say will not leave this dock."`
+			`The dockworker thinks for a minute while he pulls out another cigarette. "Alright," he finally replies, "I'll talk. But only alone, and not here - too many prying eyes, you understand." You nod, and he signals the other dockworkers to begin loading the ships while the two of you head to a small office near the docks.`
+			`Under a small, flickering light, the two of you sit down at a folding table. "Here's what happened," he finally says. "Last month, we got an order of some building materials for the base at New Iceland. Thing is, the guy they sent felt like he was entitled to the materials and gave us an earful for being, quote, 'ungrateful of their protection of Syndicate space,' unquote."`
+			`	"The Navy is only responsible for protecting the Republic - the Syndicate tends to look after itself," you comment.`
+			`	"See, that's what *I* told him. But the moment I did, he punched me right in the jaw and knocked me to the ground, telling me, 'you need to learn your place, you piece of shit.' Once we had loaded the materials into his ships, he pulled me aside and specifically told me, 'if you tell anyone of what I did, I will kill you, your family, and everyone you loved - and I have files on hand to help me.' As large as the Navy is, I was inclined to believe him."`
+			`	"I'm guessing you didn't report this to your bosses."`
+			`	"Hell, no. One, they wouldn't have done anything - most likely just brushed it off as 'he was having a bad day,' but that's the Syndicate for you. Two, they probably would've docked me a few days' pay for raising the issue - the Syndicate pays well, to be sure, but it's still just barely enough to make ends meet. And I wasn't about to call his bluff - no way am I putting my wife, my daughter and two sons in danger."`
+			`	"Can you describe him?"`
+			`	"Do you one better," he says as he stands up, walking over to a file cabinet. The drawers rattle and clunk as he pulls out a manilla-colored file with some photos inside. "I looked through security footage on the day he came through, in case I needed to 'disappear' for a bit." On a photo showing several Navy personnel walking alongside some forklifts, he points to one in particular - a face you quickly identify as Ranulph Hines. "That's him," he says, tapping Ranulph's face.`
```

Comment:
Why does a dockworker have a file cabinet, and not just like, a photo on his phone or something?